plugins/out_http/http.c

```diff
@@ -377,6 +410,21 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_out_http, http_passwd),
      "Set HTTP auth password"
     },
+#ifdef FLB_HAVE_SIGNV4
+#ifdef FLB_HAVE_AWS 
+    {
+     FLB_CONFIG_MAP_BOOL, "aws_auth", "false",
+     0, FLB_TRUE, offsetof(struct flb_out_http, has_aws_auth),
+     "Enable AWS SigV4 authentication"
+    },
+    {
+     FLB_CONFIG_MAP_STR, "aws_service", NULL,
+     0, FLB_TRUE, offsetof(struct flb_out_http, aws_service),
+     "AWS destination service code, used by SigV4 authentication"
+    },
+    FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP("aws_"),
```

Comment:
I like the idea to make the config general to help on reusing it. 
The `aws_` value looks to me like a static value and has been used several different place below.
Could we create a static field somewhere and refer to the same place so in case in the future this name is not suitable we don't need to search everywhere to change it.

Reply:
Actually this is the only place it is used. Regardless, I'll make it a #define